web/styles/dark_theme.css

```diff
@@ -373,11 +373,7 @@
     }
 
     .active_private_messages_section {
-        #private_messages_section,
-        #direct-messages-list,
-        #hide-more-direct-messages {
-            background-color: hsl(199deg 33% 46% / 20%);
-        }
+        background-color: hsl(199deg 33% 46% / 20%);
```

Comment:
A further cleanup step here would be turn these values into CSS variables.

Reply:
Started a discussion on [CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/DM.20-.20active.20narrow.20filter.20background.20color.20dark.20mode/near/1829356) for a slight visual change to make things consistent for active narrows.